templates/app/views/home/_collection.erb

```diff
@@ -1,7 +1,7 @@
 <% if products.length > 0 %>
   <section class="wrapper mb-12">
     <div class="flex items-center justify-between mb-7">
-      <h3 class="font-serif text-h4 md:text-h3"><%= title %></h3>
```

Comment:
This seems like content worth indexing.

Reply:
Absolutely, just it the context in which it's displayed.